test/linearalgebra.jl

```diff
@@ -0,0 +1,54 @@
+include("header.jl")
+using LinearAlgebra
+
+w = randn(3,2)
+wt = randn(2,3)
+wsquare = randn(3,3)
+udg = [1.,2.,3.]
+dg = [4.,5.,6.]
+ldg  = [7.,8.,9.]
```

Comment:
all of these definitions should go in the testset block to avoid polluting the global namespace. Also the functions below can be defined in the testset

Reply:
Updated